types/index.d.ts

```diff
@@ -491,6 +493,90 @@ declare module 'stripe' {
       params?: {[key: string]: unknown},
       options?: Stripe.RawRequestOptions
     ): Promise<Stripe.Response<unknown>>;
+
+    /**
+     * Parses webhook event payload into a ThinEvent and verifies webhook signature.
+     * To get more information on the event, pass the id from the returned object to
+     * `stripe.v2.core.events.retrieve()`
+     *
+     * @throws Stripe.errors.StripeSignatureVerificationError
+     */
+    parseThinEvent: (
```

Comment:
I'm having trouble understanding why this file is different than the `.d.ts` here: [stripe/stripe-node-next | Types.d.ts#L151-L166](https://github.com/stripe/stripe-node-next/blob/db30eea2a17487811daf1b5cc380bea69e1a7ce8/src/Types.d.ts#L151-L166)

It seems like we only need one or the other

Reply:
Fixed in https://github.com/stripe/stripe-node/pull/2190/commits/628b5d37d1501b0ea93eafbf546ce55420e23fc9